ldap2pg/config.py

```diff
@@ -335,6 +335,10 @@ class Configuration(dict):
             WHERE role.rolsuper IS TRUE
             ORDER BY 1;
             """.replace("\n" + ' ' * 12, "\n").strip(),
+            'schemas_query': """
+            SELECT nspname FROM pg_catalog.pg_namespace
+            ORDER BY 1;
```

Comment:
@dirks the new query don't have WHERE. However, it's now customizable through `postgres:schemas_query`.